src/bors/handlers/trybuild.rs

```diff
@@ -190,7 +193,7 @@ pub(super) async fn command_try_cancel(
     author: &GithubUser,
 ) -> anyhow::Result<()> {
     let repo = repo.as_ref();
-    if !check_try_permissions(repo, pr, author).await? {
+    if !has_try_permission(repo, author, pr, &db).await? {
```

Comment:
This now no longer sends a comment when the user doesn't have enoug privileges. And since CI is green, we probably didn't have a test for it :)